pkg/es/wrapper/wrapper.go

```diff
@@ -12,12 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package es
+package eswrapper
```

Comment:
moved these files to `wrapper` package, which makes it more obvious why coverage needs to be suppressed. The parent package is not testable (even though it only has type defs and thus an `empty_test.go` file).